src/main/java/cinema/model/ShoppingCart.java

```diff
@@ -6,8 +6,10 @@
 import javax.persistence.MapsId;
 import javax.persistence.OneToMany;
 import javax.persistence.OneToOne;
+import javax.persistence.Table;
 
 @Entity
+@Table(name = "shopping_cart")
```

Comment:
```suggestion
@Table(name = "shopping_carts")
```

add JoinTable for shopping_carts_tickets table